README.md

```diff
@@ -553,6 +553,8 @@ Services such as [CloudFlare](https://www.cloudflare.com/dns/) and [Route 53](ht
 * Latency-based
 * Geolocation-based
 
+Managed DNS Services such as  [CloudFlare](https://www.cloudflare.com/dns/) and [Route 53](https://aws.amazon.com/route53/) are very common during interviews these days. System design questions may revolve around managing mulitple availability zones of your application (e.g. like amazon.in and amazon.com) and how would you route users to their appropiate geographical location depending on their source, so emphasise on these concepts more.
```

Comment:
mulitple-> multiple

Reply:
In my experience, asking about the concepts within these popular DNS services are common, but not knowing these specific services is perfectly fine. I would phrase this a little differently.

Also, availability zones are different from different top-level domain names like .com and .in. Availability zones are within the same AWS region. So the second line needs rephrasing too.